CMakeLists.txt

```diff
@@ -89,6 +94,13 @@ if( WIN32 )
   add_definitions(-D_WIN32_WINNT=0x0601)
 endif()
 
+# Suppress warning on Travis about inconsistent missing 'override'
```

Comment:
Isn't it easier to just add the missing 'override' (or fix the inconsistency), instead of suppressing the warning?

Reply:
yes, it would be better to fix it properly, but when I simply added 'override' to SuperElastixComponent, it didn't compile in visual studio anymore! When I added the pure virtual declaration in ComponentBase, and added the override in SuperElastixComponent, then it would compile, and the warnings would disappear on Mac, but some tests started to fail. So I chose to go for a pragmatic approach and suppress the warning on Mac.

Reply:
@FBerendsen Should SuperElastixComponent::GetComponentName() always override a pure virtual function from one of its base classes (UpdateInterface or MultiStageTransformInterface?)?

Reply:
I discussed with Stefan and Kasper and in general it seems fine to me. But maybe there are still things you like to discuss or solve.

Reply:
as just discussed, Kasper and Floris previous concluded that the override should not be necessary. As both Visual Studio and GCC don't complain, we decided it's OK to suppress the warnings on Apple Clang.

Reply:
@FBerendsen Do I understand correctly that SuperElastixComponent<AcceptingInterfaces, ProvidingInterfaces> _might_ be inherited from an interface that has virtual GetComponentName() = 0? So that SuperElastixComponent::GetComponentName() _might_ override a virtual function from an interface? So does it mean that GetComponentName()=0 is not always declared in one of its interfaces?